apps/services/search-indexer/src/migrate/migrateElastic.ts

```diff
@@ -55,6 +55,17 @@ class App {
               await elastic.updateIndexTemplate(locale, esPackages)
               logger.info('updated template', { newIndexName })
               await elastic.importContentToIndex(locale, newIndexName, 'full')
+
+              const oldIndexName = await elastic.getPreviousIndex(locale)
+              if (oldIndexName) {
+                await elastic.migratePopularityScores(
+                  oldIndexName,
+                  newIndexName,
+                )
+                logger.info('Popularity scores from previous index migrated', {
+                  oldIndexName,
+                })
+              }
```

Comment:
Maybe add a info log in else here to make it clear this did not run maybe something like:
`logger.info('No older index found, skipping popularity score migration', {locale})`